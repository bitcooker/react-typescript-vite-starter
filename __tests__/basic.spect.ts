describe('basic testing', () => {
  it('snapshots an object', () => {
    expect({ state: { id: 23 } }).toMatchSnapshot()
  })
})
