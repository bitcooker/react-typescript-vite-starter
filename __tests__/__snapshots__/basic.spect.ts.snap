// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic testing snapshots an object 1`] = `
{
  "state": {
    "id": 23,
  },
}
`;
